make.js

```diff
@@ -175,6 +179,78 @@ target.bootlint = () => {
     }, 2000);
 };
 
+target.htmllint = () => {
+    const port = 3081;
+
+    env.PORT = port;
+    env.NODE_ENV = 'development';
```

Comment:
Same with this, try `process.env.NODE_ENV = 'development';`.

Reply:
I think something else is wrong. Running `node make htmllint` works fine; it breaks when bootlint is run before the htmllint target. It's like the next target starts while the old one hasn't finished yet.